### PR TITLE
update OCP support

### PR DESCRIPTION
**Pull request by antyona**
OCP support amended to say we support 4.6 and higher

### CODE REVIEW

File: docs/Content/Preparing_for_installation.md [hunk 1]
```diff
@@ -13,7 +13,7 @@ License Service is supported on all Kubernetes-orchestrated clouds on Linux x86_
 
 It was tested on the following systems:
 
-- Red Hat OpenShift Container Platform 3.11.200, 4.1, 4.2, 4.3 or newer
+- Red Hat OpenShift Container Platform 4.6 or newer
 - Kubernetes 1.11.3 or higher
```
Comment (SzymonKowalczyk) [inline docs/Content/Preparing_for_installation.md:17]: Kubernetes 1.16 or higher

**Review by SzymonKowalczyk — COMMENTED** [review]
Please also change Kubernetes client version to 1.16 or higher:

https://github.com/IBM/ibm-licensing-operator/blob/release-1.7/docs/Content/Install_from_scratch.md#L10

File: docs/Content/Preparing_for_installation.md (hunk 1 above)
Comment (antyona) [reply docs/Content/Preparing_for_installation.md]: Updated

**Review by SzymonKowalczyk — APPROVED** [review]

**Pull request merged**